prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java

```diff
@@ -104,23 +133,29 @@ private boolean channelsMissingManifest() {
         return channels.stream().anyMatch(c->c.getManifestRef() == null);
     }
 
-    private void setUpBuildEnv(List<Repository> additionalRepositories, Optional<Path> provisionConfigFile,
-                               Optional<ChannelManifestCoordinate> manifestRef, List<Channel> channels) throws IOException {
-        if (!provisionConfigFile.isPresent() && !manifestRef.isPresent()) {
-            if (!additionalRepositories.isEmpty()) {
-                this.channels = TemporaryRepositoriesHandler.addRepositories(channels, additionalRepositories);
+    // TODO: externalize
+    private static List<RemoteRepository> defaultChannelResolutionRepositories() {
```

Comment:
How about overriding (or at least supplementing) those with `overrideRepositories`?

Reply:
I was planning originally to have separate CLI option to control this, but attaching this to `--repositories` as well seems OK.